buddyboss/buddyboss-platform
# A non-existing image called undefined.png is being requested when using emoji - Fixed

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #3105 .

### How to test the changes in this Pull Request:

1. Make sure to have emoji enabled on the entire site
2. Go to the activity page
3. Click the input field for creating an activity and see non existing image is not being called anymore

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
